core/integration/features/perpetual-mark-and-index-price.feature

```diff
@@ -0,0 +1,122 @@
+    Given the parties submit the following liquidity provision:
+      | id  | party  | market id | commitment amount | fee | lp type    |
+      | lp1 | lpprov | ETH/DEC19 | 3905000000000000  | 0.3 | submission |
+    And the parties place the following pegged iceberg orders:
+      | party  | market id | peak size        | minimum visible size | side | pegged reference | volume           | offset | reference   |
+      | lpprov | ETH/DEC19 | 4000000000000000 | 3905000000000000     | buy  | BID              | 4000000000000000 | 1      | lp-ice-buy  |
+      | lpprov | ETH/DEC19 | 4000000000000000 | 3905000000000000     | sell | ASK              | 4000000000000000 | 1      | lp-ice-sell |
+    And the parties place the following orders:
+      | party   | market id | side | volume | price  | resulting trades | type       | tif     | reference |
+      | trader1 | ETH/DEC19 | buy  | 5      | 1001   | 0                | TYPE_LIMIT | TIF_GTC | t1-b-1    |
+      | trader1 | ETH/DEC19 | buy  | 5      | 900    | 0                | TYPE_LIMIT | TIF_GTC | t1-b-2    |
+      | trader1 | ETH/DEC19 | buy  | 1      | 100    | 0                | TYPE_LIMIT | TIF_GTC | t1-b-3    |
+      | trader2 | ETH/DEC19 | sell | 5      | 1200   | 0                | TYPE_LIMIT | TIF_GTC | t2-s-1    |
+      | trader2 | ETH/DEC19 | sell | 1      | 100000 | 0                | TYPE_LIMIT | TIF_GTC | t2-s-2    |
+      | trader2 | ETH/DEC19 | sell | 5      | 951    | 0                | TYPE_LIMIT | TIF_GTC | t2-s-3    |
+    When the opening auction period ends for market "ETH/DEC19"
+    Then the market data for the market "ETH/DEC19" should be:
+      | mark price | trading mode            | auction trigger             | target stake | supplied stake   | open interest |
+      | 976        | TRADING_MODE_CONTINUOUS | AUCTION_TRIGGER_UNSPECIFIED | 134907600000 | 3905000000000000 | 5             |
+    And the parties should have the following account balances:
+      | party   | asset | market id | margin       | general                   |
+      | trader1 | ETH   | ETH/DEC19 | 113402285504 | 9999999999999886597714496 |
+    And the parties should have the following margin levels:
+      | party   | market id | maintenance | search       | initial      | release      |
+      | trader1 | ETH/DEC19 | 94501904587 | 103952095045 | 113402285504 | 132302666421 |
+    
+    And the network moves ahead "2" blocks
+
+    Then the oracles broadcast data with block time signed with "0xCAFECAFE1":
+      | name           | value | time offset |
+      | perp.ETH.value | 140   | -1s         |
+
+    Then the product data for the market "ETH/DEC19" should be:
+      | internal twap     | external twap |
+      | 10500000000000000 | 140           |
```

Comment:
Why is this number correct? Can we have the corresponding calc in a comment? I am not saying it's not, I just don't see it and it.

Reply:
Thank you!